manual/manual.tex

```diff
@@ -125,32 +120,31 @@
   \begin{abstract}
 % This list of contributors is based on the hg log. If you make commits please
 % add your name in alphabetical order.
-The primary authors of this document are Matthew~Grosvenor and Adam~Walker,
+The primary authors of this document are Matthew Grosvenor and Adam Walker,
 with contributions from
-Adrian~Danis,
```

Comment:
I added this on purpose, intention was really to keep the names together, especial for people with more than two words in a name, So I'm curious why you remove it again.

Reply:
Right, I was wondering where it came from. Forcing LaTeX to keep all names together makes it impossible for it to actually lay out the paragraph correctly -- most lines go into the margin, and for some it made LaTeX try to hyphenate the name, which looked even worse.

It's fine for first/last names to not be on the same line, you'll find that pretty much everywhere. I've left `Le~Sueur` in, because even I found that strange on seperate lines.

We could change the whole thing to print each name on a separate line, which would also give it a bit more prominence.

Reply:
I agree that preventing first-last name separation is not needed. However, the overfull lines are prevented by having a \sloppy right after the \documentclass command (almost always the right thing unless you do a lot of layout copy-editing)